doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
+*%d*::
   Print the duration of the appointment in seconds
-*e*::
-  Print the end time of the appointment as UNIX time stamp
-*E*::
-  Print the end time of the appointment using the *hh:mm* format
-*m*::
+*%e*::
+  Print the end time of the appointment as the Unix time in seconds
+*%E*::
+  Print the end time of the appointment or the marker +..:..+ if
+  it ends after midnight
+*%m*::
   Print the description of the item
-*n*::
+*%n*::
   Print the name of the note file belonging to the item
-*N*::
+*%N*::
   Print the note belonging to the item
-
-Format specifiers for events
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-*m*::
+*%r*::
+  Print the remaining time before the appointment
+*%s*::
+  Print the start time of the appointment as the Unix time in seconds
+*%S*::
+  Print the start time of the appointment or the marker +..:..+ if it begins
+  before midnight
+
+.+events+
+
+[horizontal]
+*%m*::
   Print the description of the item
-*n*::
+*%n*::
   Print the name of the note file belonging to the item
-*N*::
+*%N*::
   Print the note belonging to the item
 
-Format specifiers for todo items
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.+todo items+
 
-*p*::
-  Print the priority of the item
-*m*::
+[horizontal]
+*%m*::
   Print the description of the item
-*n*::
+*%n*::
   Print the name of the note file belonging to the item
-*N*::
+*%N*::
   Print the note belonging to the item
+*%p*::
+  Print the priority of the item
+
+[[_default_format_strings]]
+.Default format strings
+
+For each format option there is a default format string which is used when the option
+is not given. In query results the default format options are:
+
+  --format-apt " - %S -> %E\n\t%m\n"
+  --format-recur-apt " - %S -> %E\n\t%m\n"
+  --format-event " * %m\n"
+  --format-recur-event " * %m\n"
+  --format-todo "%p. %m\n"
+
+In all other cases (grep and dump-imported) the default format string is +"%(raw)"+.
+
+[[_extended_format_specifiers]]
+.Extended format specifiers
+
+Extended format specifiers can be used to control the printing of times for
+some of the single-letter specifiers. Additionally there are two specifiers
+that do not have any corresponding short form and are intended for use in
+scripting.
+
+*%d* ~ *%(duration*`[`*:*'format'`]`*)*::
+*%r* ~ *%(remaining*`[`*:*'format'`]`*)*::
+  'format' may contain any of the strftime(3) specifiers *%d*, *%H*, *%M* or
+  *%S* with the following modifications: 1) days are not limited to the
+  "calendar" values 0-31 (hours, minutes and seconds are "clock" values, but
+  see *E* in the following) 2) each number is by default padded with zeros to
+  two decimal places, 3) the *%* character may be followed by one or two
+  optional flags: *-*, which suppresses the zero-padding, *E*, which will
+  suppress the "clock" limits on *%H*, *%M* and *%S*; if both are used, *-*
+  must precede *E*, 4) no other flags or width specifications are supported
+
+*%s* ~ *%(start*`[`*:*'format'`]`*)*::
+*%e* ~ *%(end*`[`*:*'format'`]`*)*::
```

Comment:
Same here. I think we can simply drop `*%s* ~` and `*%e* ~` here since they are already mentioned in the text below.